Jenkinsfile_nightly

```diff
@@ -12,11 +12,20 @@ def product = "fpl"
 def component = "fpl-service"
 
 withNightlyPipeline(type, product, component) {
-  enableMutationTest()
 
-  after('mutationTest') {
-    withSonarQubeEnv("SonarQube") {
-      sh "./gradlew --info sonarqube"
-    }
-  }
+  env.URL = 'http://ccd-case-management-web-aat.service.core-compute-aat.internal';
+  env.PROXY_SERVER = "proxyout.reform.hmcts.net:8080";
+
+//  enableMutationTest()
+//
+//  after('mutationTest') {
+//    withSonarQubeEnv("SonarQube") {
+//      sh "./gradlew --info sonarqube"
+//    }
+//  }
+
+  //to Run functional test
+  sh "./gradlew runFunctionalTest"
```

Comment:
I do not think it would even run TBH. It is definite not what the pipeline should look like.

Reply:
Also calling `runFunctionalTest` itself would not work as there would not be yarn dependencies installed. A `functional` task is an entry point to functional tests.

Reply:
Hi Damian, changed to functional task as an entry point.